.github/workflows/proto-docker.yml

```diff
@@ -31,16 +34,28 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2 
+        if: ${{ github.event_name != 'pull_request' }}
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          
+      - name: Build
+        uses: docker/build-push-action@v2
+        if: ${{ github.event_name == 'pull_request' }}
         with:
-          username: ${{ secrets.DOCKERHUBTM_USERNAME }}
-          password: ${{ secrets.DOCKERHUBTM_TOKEN }}
+          file: contrib/devtools/Dockerfile
+          context: ./contrib/devtools
+          platforms: linux/amd64,linux/arm64
+          push: false
 
-      - name: Publish to Docker Hub
+      - name: Publish to GHCR
         uses: docker/build-push-action@v3
         with:
           context: ./contrib/devtools
           platforms: linux/amd64,linux/arm64
           push: true
```

Comment:
```suggestion
          push: ${{ github.event_name != 'pull_request' }}
```
why not do this?

Reply:
also there will need to be a followup pr to update the make file

Reply:
Your way is better.  Should do that.